demo/events.js

```diff
@@ -46,6 +46,10 @@ function addEvents(grid, id) {
     console.log(g + 'resizestart ' + el.textContent + ' size: (' + w + ' x ' + h + ')');
   });
 
+  grid.on('resize', function(event, newWidget) {

```

Comment:
make it el to be consistent with others and pass the DOM element, not the node...